### ISSUE TITLE
Strict adherence to gitignore format on forceignore incorrectly calls out Default_Navigation.navigationMenu-meta.xml

### ISSUE THREAD

**Issue by ImJohnMDaniel** [forcedotcom/cli#772]
### Summary

The CLI is now announcing a message that with Spring21, the `.forceignore` must strictly adhere to `.gitignore` format.

When working with communities and ExperienceBundle metadata format, the file `Default_Navigation.navigationMenu-meta.xml` could appear.  We are forceignoring this in order to not add confusion to our project navigationMenu files.

The CLI is warning that `Default_Navigation.navigationMenu-meta.xml` is not in the correct format.

### Steps To Reproduce:

1.  create a scratch org with communities enabled
2. Add `Default_Navigation.navigationMenu-meta.xml` to the `.forceignore` file
3. Execute `sfdx force:source:pull` command

### Expected result

No warning message

### Actual result

Warning message seen
```
Warning: The .forceignore file doesn't adhere to .gitignore format which will be the default behavior starting in Spring '21 release. More information on .gitignore format here: https://git-scm.com/docs/gitignore. Fix the following lines in your .forceignore and add '# .forceignore v2' to your .forceignore file to switch to the new behavior.
(node:51273) Warning:   Default_Navigation.navigationMenu-meta.xml
```

### Additional information

**SFDX CLI Version**(to find the version of the CLI engine run sfdx --version):
```
sfdx-cli/7.82.0-3d0c527ac6 darwin-x64 node-v12.18.3
```

**SFDX plugin Version**(to find the version of the CLI plugin run sfdx plugins --core)
```
@dx-cli-toolbox/sfdx-toolbox-package-utils 0.3.3 (link) /Users/john/workspace/_cli-related/sfdx-toolbox-package-utils
@oclif/plugin-autocomplete 0.3.0
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.2.0 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.9.4 (core)
@oclif/plugin-update 1.3.10 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-diff 0.0.6
@salesforce/sfdx-trust 3.4.3 (core)
alias 1.1.3 (core)
analytics 1.12.1 (core)
auth 1.3.0 (core)
config 1.2.1 (core)
generator 1.1.3 (core)
salesforcedx 50.7.0 (core)
├─ schema 1.0.1 (core)
├─ templates 50.1.0 (core)
├─ salesforce-alm 50.7.0 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
├─ apex 0.1.4 (core)
└─ custom-metadata 1.0.10 (core)
sfdmu 3.7.21
sfdx-cli 7.82.0 (core)
shane-sfdx-plugins 4.43.0
├─ @mshanemc/plugin-streaming 1.1.7
└─ @mshanemc/sfdx-sosl 1.1.0
soqlx-opener 0.1.12
```

**OS and version**:
MacOS v11.0.1

**Comment by jclark-dot-org** [comment]
I'm seeing the same issue with the line:
```
LightningSalesConsole_UtilityBar.flexipage-meta.xml
```

I've tried backslash in front of `_`, `-`, and both, no change.

Mac OS 10.14.5

```
DX Version:
sfdx-cli/7.82.0-3d0c527ac6 darwin-x64 node-v12.18.3

Plugin Versions:
@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.2.0 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.9.4 (core)
@oclif/plugin-update 1.3.10 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.4.3 (core)
alias 1.1.3 (core)
analytics 1.12.1 (core)
auth 1.3.0 (core)
config 1.2.1 (core)
generator 1.1.3 (core)
salesforcedx 50.7.0 (core)
├─ schema 1.0.1 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
├─ apex 0.1.4 (core)
├─ salesforce-alm 50.7.0 (core)
├─ custom-metadata 1.0.10 (core)
└─ templates 50.1.0 (core)
sfdx-cli 7.82.0 (core)
sfpowerkit 2.4.13
```

**Comment by mshanemc** [comment]
this is no longer reproducible for me (probably due to the changes to forceignore) on either source:pull or the new source:beta:pull.

I'll close it, and please open a new issue if you can reproduce it (that way we get updated plugins/version info, too)
